### ISSUE TITLE
Make work with Github and Facebook: Content Security Policy

### ISSUE THREAD

**Issue by joereddington** [joereddington/PrivateInPublic#5]
Github gives the following error: 

```

VM807:1 Refused to load the script 'https://cdnjs.cloudflare.com/ajax/libs/crypto-js/3.1.2/rollups/aes.js' because it violates the following Content Security Policy directive: "script-src assets-cdn.github.com".

(anonymous) @ VM807:1
(anonymous) @ VM807:1

```

This: https://github.com/twbs/bootlint/issues/201

suggests that there isn't much we can do...

**Comment by joereddington** [comment]
Closed by #6 - the bookmarklet WON'T work in some sites but the extension will. Need to update the Readme with that.

**Comment by SecureCloud-biz** [comment]
??
